java/lib/src/main/java/com/svix/MessageAttempt.java

```diff
@@ -78,24 +78,23 @@ public void resend(final String msgId, final String appId, final String endpoint
 
 	public void resend(final String msgId, final String appId, final String endpointId, final PostOptions options) throws ApiException {
```

Comment:
Wow, our SDK is terrible here! It should be app id -> msg id -> endpoint id.

Reply:
Let's do that in a followup PR?